Signal/src/util/OWSBackupAPI.swift

```diff
@@ -40,9 +54,9 @@ import CloudKit
                                                        failure: @escaping (Error) -> Void) {
         saveFileToCloud(fileUrl: fileUrl,
                         recordName: "ephemeralFile-\(NSUUID().uuidString)",
-                        recordType: signalBackupRecordType,
-                        success: success,
-                        failure: failure)
+            recordType: signalBackupRecordType,
```

Comment:
The thrust of the changes in this class are to:

* Add simple (not exponential backoff) retry.
* Improve handling of edge cases.
* DRY where possible.